### PR TITLE
Reworked OBR-7 and OBR-8 mapping requirements.

### PR DESCRIPTION
**Pull request by PeterKranich**
## 📑 Description
Reworked OBR-7 and OBR-8 mapping requirements.

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed
- [ ] I have selected a committee co-chair to review the PR

## ℹ Additional Information
None

### CODE REVIEW

**Review by riechkathrin — APPROVED** [review]
You may wait for a review from @MartinKasp

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-dec.adoc
Comment (MartinKasp) [inline asciidoc/volume2/gateways/tf2-ch-b-gateway-dec.adoc:316]: Proposed extension/clarification:
"For continuously (periodically) measured metrics, a SOMDS DEC Gateway shall [...]"

Comment (PeterKranich) [reply asciidoc/volume2/gateways/tf2-ch-b-gateway-dec.adoc:316]: The start date and time of the current export interval is independent of the metric measurement type. The interval is defined by the gatewasy and also the eperiodic data are exported in the interval but with the measurment time set in the OBX-14 field.

**Pull request merged**
